README.md

```diff
@@ -41,6 +41,8 @@ m6_web_daemon:
 
 ## Write command
 
+This command use the [event-loop component](https://github.com/reactphp/event-loop#usage) that uses [ReactPHP](https://reactphp.org) for execute loops; and other tasks asynchronously.
```

Comment:
> that uses [ReactPHP](https://reactphp.org) for execute loops; and other tasks asynchronously.

=> which [ReactPHP](https://reactphp.org) uses to run loops and other asynchronous tasks.


(either `run` or `make` would be good to me)